### DISCUSSION TITLE
Sport Category

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#423] [Q&A] [answered]
Hi.  I see that recently some TV channels have stopped working in the sports category such as Thai True Premier Football and Portuguese Sport TV.  For me, none of those channels work and it is not written that it is geo blocked.  Is the same the case with you?
Also I'm new to this IPTV so I don't really know everything about links.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream
